indico/modules/search/client/js/components/SearchApp.jsx

```diff
@@ -298,6 +298,7 @@ export default function SearchApp({category, eventId, isAdmin}) {
           searchTerm={q || ''}
           placeholders={eventSearch ? [] : options?.placeholders || []}
         />
+        {renderPluginComponents('search-app-below-input', {user: Indico.User})}
```

Comment:
You do not need to pass the user here, you can use it directly in your plugin.
Also, I suggest passing the props such as category, eventId and possibly isAdmin instead since especially the first two would be much harder to get from plugin code (even if you do not need those).

Reply:
Fixed.